app/index.js

```diff
@@ -247,7 +248,13 @@ module.exports = generators.Base.extend({
       }
     }
   },
-  install: function () {
-    if(!this.options['skip-install']) this.installDependencies();
+  install: function() {
+    if (!this.options['skip-install']) {
+      return this.installDependencies(function() {
+        return spawn('npm', ['run', 'test:coverage'], {
```

Comment:
After testing I don't think always running tests after an install is good. If a DB isn't setup or this isn't the actual environment it will run in it will always fail, and isn't a good thing to see as soon as you've setup your project. I'm moving this to be only for those `test:generate` scripts